etc/kayobe/ansible/pulp-image-upload.yml

```diff
@@ -7,15 +7,16 @@
     remote_pulp_password: "{{ stackhpc_image_repository_password }}"
     repository_name: "kayobe-images-{{ openstack_release }}-{{ os_distribution }}-{{ os_release }}"
     pulp_base_path: "kayobe-images/{{ openstack_release }}/{{ os_distribution }}/{{ os_release }}"
```

Comment:
In general, we may want to customise this path a bit more, I think it might need one more variable at the start so you can change `kayobe-images` to instead be something like `ipai-images` for example

Reply:
Setting the image_name extra var to `kayobe-images` in the host image build. Set to `ipa` in the IPA workflow.